graphql-api/src/queries/mitochondrial-coverage-queries.ts

```diff
@@ -7,6 +7,7 @@ import { extendRegions, mergeOverlappingRegions, totalRegionSize } from './helpe
 import { assertDatasetAndReferenceGenomeMatch } from './helpers/validation-helpers'
 
 const COVERAGE_INDICES = {
+  gnomad_r4: 'gnomad_v3_mitochondrial_coverage',
```

Comment:
This makes complete sense, especially right now -- just commenting to say this is maybe something to mentally flag for the future. Coming back to this without context could be confusing.